manage/trigger/how-do-they-work.rst

```diff
@@ -14,33 +14,47 @@ In some situations triggers might be the wrong choice, see
    Please have a look at :doc:`/channels/email/email-headers`
    in case this is a relevant use case for you.
 
+Activators
+----------
+
+Triggers support two types of activators:
+
+* **Action** is triggered by some actor.
+  Either a user creating or updating the ticket. Or an external action. For example an email coming in.
+
+* **Time event** is triggered at a specific time.
+  E.g. when Ticket's Pending time is reached.
+
+
+**Action** activator may evaluate **Conditions** in two modes:
+
+* **Selective** execution checks if any property that is included in conditions was updated.
+  If the action was other than update, it checks if conditions match.
+
+  .. important::
+    E.g. A selective action trigger which is listening on the priority **1 low** will trigger if the
+    ticket was changed to **1 low** or got an new article in that priority state.
+
+* **Always** execution checks if the current state of the ticket in question matches conditions.
```

Comment:
Please add _mode_ next to _execution_, it has an entirely different meaning when used as a proper noun 💀